python/cells.cpp

```diff
@@ -427,88 +394,85 @@ void register_cells(pybind11::module& m) {
           "`proportion` is the proportion of the CV (itegrated by area or length) included in the region."
     );
 
-    pybind11::class_<arb::init_membrane_potential> membrane_potential(m, "membrane_potential", "Setting the initial membrane voltage.");
     membrane_potential
-        .def(pybind11::init([](double v) -> arb::init_membrane_potential { return {v}; }))
+        .def(py::init([](const U::quantity& v,
+                         std::optional<std::string> s) -> arb::init_membrane_potential {
+            return {v };
```

Comment:
why do we need the optional string?

Reply:
Since we should parse it into a iexpr, of course :P

Reply:
The rules are that you cannot add a scale to a default -- have been ever since we allowed scales. I propose removing the write access via these newtype wrapper, since we already have an interface for doing. Offering
two with different policies is just causing confusion.